src/lib/pickles/plonk_checks/scalars.ml

```diff
@@ -2585,12 +3948,16 @@ module Tick : S = struct
                          * field
                              "0x0000000000000000000000000000000000000000000000000000000000001000"
                          * cell (var (Witness 2, Curr))
-                       - cell (var (Witness 0, Curr)) ) ) ) )
+                       - cell (var (Witness 0, Curr)) ) )
+               , fun () ->
+                   field
+                     "0x0000000000000000000000000000000000000000000000000000000000000000"
+               ) ) )
       ; ( Index ForeignFieldAdd
         , lazy
-            (enabled_if
+            (if_feature
```

Comment:
Mmmm.... But there are no lookups in the ForeignFieldAdd gate... Here I am pasting the part of rust code that assigns a pattern to each of the gate types:
```
        match (gate_type, curr_or_next) {
            (ChaCha0 | ChaCha1 | ChaCha2, Curr | Next) => Some(LookupPattern::Xor),
            (ChaChaFinal, Curr | Next) => Some(LookupPattern::ChaChaFinal),
            (Lookup, Curr) => Some(LookupPattern::Lookup),
            (RangeCheck0, Curr) | (RangeCheck1, Curr | Next) | (Rot64, Curr) => {
                Some(LookupPattern::RangeCheck)
            }
            (ForeignFieldMul, Curr) => Some(LookupPattern::ForeignFieldMul),
            (Xor16, Curr) => Some(LookupPattern::Xor),
            _ => None,
        }
```

Reply:
I think this whole part should be gone